Description: node-environment-flags isn't required anywhere
 and it's just a polyfill of process.allowedNodeEnvironmentFlags
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/977232
Forwarded: no
Last-Update: 2022-09-15

--- a/packages/babel-node/package.json
+++ b/packages/babel-node/package.json
@@ -26,7 +26,6 @@
     "@babel/register": "workspace:^",
     "commander": "^4.0.1",
     "core-js": "^3.26.0",
-    "node-environment-flags": "^1.0.5",
     "regenerator-runtime": "^0.13.11",
     "v8flags": "^3.1.1"
   },
